### DISCUSSION TITLE
Packages kept back in Ubuntu apt upgrade

### DISCUSSION THREAD

**Discussion by jhdore** [Security-Onion-Solutions/securityonion#7621] [Unsupported Versions] [answered]
Hi all,

I have recently deployed SecurityOnion on an Ubuntu 20.04 LTS server, and it is now reporting five packages are being kept back from updates when I do sudo apt upgrade - 

```
console@bun-onion:~$ sudo apt upgrade 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Calculating upgrade... Done
The following packages have been kept back:
containerd.io docker-ce docker-ce-cli docker-ce-rootless-extras wazuh-agent
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
```
- is this necessarily a problem? Will these packages (containerd.io docker-ce docker-ce-cli docker-ce-rootless-extras wazuh-agent) be upgraded via soup at some point when SO releases newer versions? 

Cheers,
J

**Comment by dougburks** [comment, marked as answer]
This is not a problem. Held packages will be updated when necessary.

**Reply by jhdore** [reply]
Thank you, good to know!
